libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -901,6 +902,7 @@ void NavEKF2_core::readRngBcnData()
     rngBcnDataToFuse = storedRangeBeacon.recall(rngBcnDataDelayed,imuDataDelayed.time_ms);
```

Comment:
could we make this const bool when beacon not enabled?

Reply:
Done